src/components/common/messaging/attachments/Attachment.module.scss

```diff
@@ -92,4 +92,10 @@
 
 .image {
     cursor: pointer;
+    width: 100%;
+    height: 100%;
```

Comment:
Thanks to this, I have unintentionally fixed the bug of the image not loading when it is partially visible

Reply:
Oh nice